x-pack/legacy/plugins/epm/public/types.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export enum InstallStatus {
+  installed = 'installed',
```

Comment:
I wish we could link these values to the `InstallationStatus` they're mirroring but I think that's a limitation of using `enum` / string enums. It's not easy (possible?) to extend/compose enums like you can with other types.

Can you add a comment about the relationship to `InstallationStatus` and we can come back to the typing later?

Reply:
Yea, exactly.  I tried and couldn't figure out a way around combining them as enums.  I'm not sure if InstallationStatus types could potentially change, though.  So I thought perhaps it was best to separate them from the client types, anyway.  Am happy to combine them if we want to move away from the enums again. Will add a comment.